climada/entity/impact_funcs/base.py

```diff
@@ -127,4 +127,65 @@ def check(self):
                            'origin of the intensity scale. In impact.calc '
                            'the impact is always null at intensity = 0.')
 
+    def set_step_ImpF(self, threshold, inten_min, inten_max, if_id=1):
+
+        """ Step function type impact function. Everything is destroyed above
+        threshold. Usefull for high resolution modelling.
+        
+        This method modifies self (climada.entity.impact_funcs instance)
+        by assining an id, intensity, mdd and paa to the impact function.
+        
+        Parameters
+        ----------
+            threshold : float
+                threshold over which exposure is fully destroyed
+            inten_min : float
+                minimum value of intensity range
+            inten_min : float
+                maximum value of intensity range
+            if_id : int, optional, default=1
+                impact function id
+
+        """
 
+        self.id = if_id
+        self.intensity = np.array([inten_min, threshold, threshold, inten_max])
+        self.paa = np.array([1, 1, 1, 1])
+        self.mdd = np.array([0, 0, 1, 1])
+
+    def set_sigmoid_ImpF(self, sig_mid, sig_shape, sig_max,
```

Comment:
I do not understand the name `sig_shape`. Why not simply use `k`, the exponent of the sigmoid?

Reply:
I also do not understand the sig_mid, sig_shape, sig_max. Usually, like in the TC sigmoids, we label the defining params sth like  v_min, v_half, v_max, (not exactly sure what now, but more or less)? so maybe call them thresh_...?
Especially the sig_mid as "intercept" i don't understand from the description.

Reply:
as discussed - I don't want v_* as parameters as we want to keep this impact functions as generic as possible and v has always a windspeed touch to it. Also, this sigmoid is not the Emanuel function thus I don't want to use "his" labeling of params.
But obviously it's confusing as it is now so I try to improve ;)

Reply:
ok I renamed the params to L, k, and x0 to be in line with the wiki definition of the sigmoid. also, I've included the function in the docstring for "easier" understanding:
.. math::
            f(x) = \frac{L}{1+exp^{-k(x-x0)}}
.. and adjusted accordingly in 944f9aa7